### ISSUE TITLE
Proper Filenames for SEO instead of an ID

### ISSUE THREAD

**Issue by ghost** [alanpich/tvImagePlus#66]
I recently started testing you're package in some of my projects. Is there any chance to use the original path/filename instead in combination with an unique ID instead of merely an ID. Search engines heart proper filenames :)

**Comment by alanpich** [comment]
Wouldn't be a problem to implement sure... do search engines really care about file names of images included on a page?

**Comment by ghost** [comment]
Yep! Google loves proper (descriptive) filenames for showing in their search results. There are many articles about the subject. img src="i/pigeon-sitting-on-a-park-bench" alt="grey pigeon sitting on a bench in the park"  is giving google a lot of useful information.
